public/client/TracyCallstack.cpp

```diff
@@ -157,6 +170,65 @@ void InitCallstack()
     SymInitialize( GetCurrentProcess(), nullptr, true );
     SymSetOptions( SYMOPT_LOAD_LINES );
 
+#ifdef TRACY_DBGHELP_LOCK
+    DBGHELP_UNLOCK;
+#endif
+}
+
+DWORD64 dbgHelpLoadSymbolsForModule(PCSTR imageName, DWORD64 baseOfDll, DWORD bllSize)
+{
+    if( !s_doOfflineSymbolResolve )
+    {
+        return SymLoadModuleEx( GetCurrentProcess(), nullptr, imageName, nullptr, baseOfDll, bllSize, nullptr, 0 );
+    }
+    return 0x0;
+}
+
+ModuleCache* dbgHelpLoadSymbolsForModuleAndCache(PCSTR imageName, DWORD imageNameLength, DWORD64 baseOfDll, DWORD dllSize)
+{
+    dbgHelpLoadSymbolsForModule( imageName, baseOfDll, dllSize );
+
+    ModuleCache* cachedModule = s_modCache->push_next();
+    cachedModule->start = baseOfDll;
+    cachedModule->end = baseOfDll + dllSize;
+
+    // when doing offline symbol resolution, we must store the full path of the dll for the resolving to work 
+    if( s_doOfflineSymbolResolve )
+    {
+        cachedModule->name = (char*)tracy_malloc_fast(imageNameLength + 2);
```

Comment:
Why is there need for two additional characters?

Reply:
you are right, it was a copy paste from the code below - fixed